relay-server/src/processing/logs/store.rs

```diff
@@ -39,12 +40,18 @@ pub struct Context {
     pub scoping: Scoping,
     /// Storage retention in days.
     pub retention: Option<u16>,
+    /// Enables serialization of log metadata into attributes.
+    pub meta_attrs: bool,
```

Comment:
I would suggest calling this `enable_meta_attrs` or `serialize_meta_attrs` (basically a more boolean-sounding name).